themes/default/content/docs/pulumi-cloud/deployments/drift.md

```diff
@@ -12,35 +12,35 @@ menu:
 
 Drift Detection is the process of identifying changes in the actual state of your cloud environment that deviate from the expected state stored in Pulumi Cloud. This deviation can occur for various reasons, including manual adjustments made directly in the cloud provider’s console, unintended consequences of scripts, or unauthorized changes.
 
-In order to detect and/or remediate drift via Pulumi Deployments, you are first required to configure deployment settings for your stack.
+To detect and remediate drift via Pulumi Deployments, you are first required to configure deployment settings for your stack.
 
-You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the pulumi program as the source of truth and override what is in the cloud.
+You can also run a remediate drift operation, which will run a `pulumi up --refresh` to treat the Pulumi program as the source of truth and override what is in the cloud.
 
-## Runing Drift detection from the CLI
+## Running Drift Detection from the CLI
 
-Any Preview of a Refresh is considered a drift detection run in Pulumi Cloud. Running drift detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
+Any preview of a refresh is considered a drift detection run in Pulumi Cloud. Running Drift Detection from the CLI is as simple as running `pulumi refresh --preview-only` or even just `pulumi refresh`, which will first run a preview (thereby creating the drift run), before running the actual refresh.
 
 After your run completes you can see the drift run via the [Drift tab](#drift-tab) for your stack.
 
 ## Pulumi Cloud UI
 
-### Running via Click to Deploy
+### Running via click to deploy
 
 You can run a drift or remediate drift run ad hoc using the Click to Deploy option from your stack.
 
 ![Click to Deploy](../click-to-deploy-drift.png)
 
-### Drift Tab
+### Drift tab
 
 We've added a new Drift tab to the stack page. If the stack is currently in a drifted state, a warning bell icon will appear on the tab.
 
 Regardless of how a drift run is initiated, all results will appear in this tab. If there is drift detected for a run, a diff summary of the changes will be included in the information card.
 
 ![Drift tab](../drift-tab.png)
 
-### Configuring Drift Detection on a Schedule
+### Configuring Drift Detection on a schedule
 
-In order to set up Drift Detection and Remediation in the Pulumi Cloud console, follow these steps:
+In order to set up Drift Detection and remediation in the Pulumi Cloud console, follow these steps:
```

Comment:
if we leave Drift Detection capitalized then Remediation should be capitalized, it we lower case remediation we should change to lower case drift detection.